pkgs/applications/science/biology/last/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, unzip, zlib, python3, parallel }:
+
+stdenv.mkDerivation rec {
+  pname = "last";
+  version = "1042";
+
+  src = fetchurl {
+    url = "http://last.cbrc.jp/last-${version}.zip";
+    sha256 = "0mgbhd01m9riqza2gx56qk6x5682kg71zflhisr6d3q05wrv103f";
+  };
+
+  nativeBuildInputs = [ unzip ];
+  buildInputs = [ zlib python3 ];
+
+  installPhase = "make install prefix=$out";
```

```suggestion
  makeFlags = [ "prefix=$out" ];
```

---

This didn't work, the $ was escaped so $out didn't expand. Is there a way to prevent escaping?

---

placeholder builtin is meant to handle this
```suggestion
  makeFlags = [ "prefix=${placeholder "out"}" ];
```

---

Thanks!